src/js/pyodide.ts

```diff
@@ -211,8 +212,15 @@ export async function loadPyodide(config: {
   if ((loadPyodide as any).inProgress) {
     throw new Error("Pyodide is already loading.");
   }
-  if (!config.indexURL) {
-    throw new Error("Please provide indexURL parameter to loadPyodide");
+  if (!options.indexURL) {
+    let err;
+    try {
+      throw new Error();
+    } catch (e) {
+      err = e;
+    }
+    const fileName = ErrorStackParser.parse(err)[0].fileName!;
```

Comment:
This is well... really hacky. Can't [`document.currentScript`](https://developer.mozilla.org/en-US/docs/Web/API/Document/currentScript) or [`import.meta.url`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import.meta) be an option?